### DISCUSSION TITLE
Would anyone know how what style is to be modified to remove the large spaces between collapsible results?

### DISCUSSION THREAD

**Discussion by tedtninety** [microsoft-search/pnp-modern-search#2761] [Q&A]
Would anyone know how what style is to be modified to remove the large spaces between collapsible results?
![collaps](https://user-images.githubusercontent.com/115969035/217870429-6ae3b139-42de-49a0-8558-0a1b9599e6cc.png)

On a results page with myriad returns, the user is forced to do a lot of unnecessary scrolling.

Thank you,
ted

**Comment by wobba** [comment]
Which template are you using to display your results?

**Comment by tedtninety** [comment]
Hi Wobba!
Thank you for responding.

I am using the Custom Template with collapsible.
However, I believe I have isolated it and found the answer..
It is .pnp-modern-search .placeholder--defaultlist .template--listitem, pnp-modern search . template--defaultlist . template -- listItem.

Change  min-height to 0px and padding to 0;
